cypress/endToEnd/i-can-view-a-legacy-beacon.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { iHavePreviouslyRegisteredALegacyBeacon } from "../common/i-have-a-legacy-beacon.spec";
+import { givenIHaveSignedIn } from "../common/selectors-and-assertions.spec";
+import { legacyBeaconRequest } from "../fixtures/legacyBeaconRequest";
+
+describe("As an account holder", () => {
+  it("I can view a legacy beacon linked to my email", () => {
+    givenIHaveSignedIn();
+    iHavePreviouslyRegisteredALegacyBeacon(legacyBeaconRequest);
+    cy.visit("/account/your-beacon-registry-account");
```

Comment:
Just add a couple of assertions here: 
- `iCanSeeTheLegacyBeaconThatIsAssociatedWithMyEmailAddress()` (or similar, play around with the wording to find what feels right)

Could you add a separate test (new `it` block) that uses both `iHavePreviouslyRegisteredALegacyBeacon` and `IHavePreviouslyRegisteredABeacon` to seed the database with both a "new" beacon and a "legacy" beacon, and asserts that they're displayed in the right place?